boards/arm/stm32l476g_disco/stm32l476g_disco.yaml

```diff
@@ -10,3 +10,4 @@ ram: 96
 flash: 1024
 supported:
   - gpio
+  - rtc
```

Comment:
should this also have counter ? (same for other board yaml)

Reply:
I think this is the limit of having one API for multiple purposes.
Do we list IPs or API here?

Reply:
ok, will add